skipchain/struct.go

```diff
@@ -553,6 +578,16 @@ func (sbs Proof) verifyChain() error {
 	return nil
 }
 
+// Signature schemes should be ordered by robustness such that for any
```

Comment:
Why? What if we want to experiment with a new sig scheme that is much cheaper, but also less secure, and so is better for some IoT use-case or whatever?

Reply:
That comment was actually anticipating the moment were we allow signature schemes to evolve for a given skipchain so that you can verify that a request to change only increase the security and it is not a malicious node trying to get a scheme known to be weak.

But yeah your comment makes totally sense. So the problem is that if we give the ordering/something else to prevent downgrade attacks, we might end up in a dead-end later if we want to change the signature schemes. I need to think about it...

Reply:
One solution I think of is to ignore the indices ordering/whatever and use a per index table to assign authorized changes.